### DISCUSSION TITLE
Preenchimento da NF-e para recebimentos do exterior

### DISCUSSION THREAD

**Discussion by marceloquinze** [DyegoCosta/trabalhando-remoto#170] [General]
Olá. Eu recebo pagamentos do exterior e fazia isso como MEI até ano passado. Como extrapolei a faixa do MEI em mais de 20%, vou ter que abrir uma ME (Simples Nacional). Eu gerava notas fiscais apenas para meu controle, mas agora vou ter que fazer tudo direitinho.
Uma dúvida que tenho é como gerar a nota fiscal da forma correta para os recebimentos do exterior. Procurei aqui nas issues e não achei. Atualmente eu recebo via Payoneer e Paypal. Eu sei que dá pra criar uma conta PJ nas duas plataformas e enviar as remessas em reais para minha conta PJ (que eu ainda tenho que abrir). 
Até aí, ok. Mas, como é que vocês informam o tomador de serviço no sistema municipal que gera a NF? Qual CNPJ deve ser informado no caso de recebimento de pagamentos do exterior via Payoneer ou Paypal? 
Valeu!

**Comment by endel** [comment]
Fala @marceloquinze, irado que seu business ta rolando!

Como você agora vai operar como Pessoa Jurídica, você terá que esquecer a pessoa física para todas as operações financeiras (menos o seu próprio pagamento, a.k.a. pró-labore, melhor vc consultar um contador para ter uma melhor noção dos impostos disso)

Sugiro criar conta com suas informações da PJ em ambas Payoneer e PayPal. Além de, é claro, a sua conta PJ no banco. Eu tenho gostado de utilizar o Remessas Online para receber do exterior diretamente na minha conta PJ, sem Payoneer ou PayPal envolvidos. (PayPal tem taxas muito altas! Payoneer é mais justo até onde sei)

Quando for emitir nota fiscal no sistema municipal, eu nunca preencho CNPJ, pois o contratante é estrangeiro. Eu sempre utilizo o endereço fiscal do contratante, e seleciono que a nota fiscal "não possui tributação no município" (seguindo orientação do meu contador)

Espero que tenha ajudado, e muito sucesso para você!

**Comment by marceloquinze** [comment]
Oi, @endel ! Muito obrigado pelo desejo de sucesso! Vamos que vamos...
Então, no meu caso eu sou meio que obrigado a receber por essas duas plataformas porque as empresas que me fazem o pagamento só trabalham com elas. Não tenho como escolher...
Eu estou iniciando com um novo contador e ele me disse que não era correto não informar o tomador de serviços na nota fiscal. Daí a minha dúvida... Aliás, sempre foi a minha dúvida. Se eu passo as informações da empresa estrangeira na nota fiscal (daí sim, claro, não vai ter CNPJ), ou se eu posso considerar a plataforma de pagamento por onde eu recebo esse dinheiro em reais, e daí eu informo ela como tomador de serviço na nota fiscal.
Poxa, eu nunca achei essa informação em lugar nenhum pra me dar segurança.

**Comment by yuriploc** [comment]
> Se eu passo as informações da empresa estrangeira na nota fiscal (daí sim, claro, não vai ter CNPJ), ou se eu posso considerar a plataforma de pagamento por onde eu recebo esse dinheiro em reais, e daí eu informo ela como tomador de serviço na nota fiscal.

Uma prática comum (mas que não vejo fontes oficiais confirmando) é usar o nome e o endereço da empresa na NFS-e, mesmo que não exista CNPJ. Porque caso algum dia você precise prestar contas com o Leão, um contrato de trabalho entre o seu CNPJ e a empresa estrangeira e as notas ficais com o nome da empresa no campo de tomador de serviços facilita bastante uma auditoria.

**Comment by lucasalmeida92** [comment]
Quanto a opção de `tributação fora do município` (que o @endel falou), tinha essa duvida tambem!

>  e seleciono que a nota fiscal "não possui tributação no município" (seguindo orientação do meu contador)

Vou abrir um chamado na Contabilizei pra confirmar.

**Comment by lucasalmeida92** [comment]
No caso nessa parte aqui:
![image](https://user-images.githubusercontent.com/5767871/159719393-9a202f99-9e46-493c-acec-7869a4cb006c.png)

**Comment by jocieldo** [comment]
Fui informado pela contabilizei para gerar dessa forma.

![image](https://user-images.githubusercontent.com/4689721/166292531-93c82e2c-f70f-4b69-b2a8-436bfa755643.png)
